Data-tranformations/fairgenomes/samples.yarrml.yaml

```diff
@@ -0,0 +1,22 @@
+prefixes:
+  ega.dataset: https://www.ebi.ac.uk/ega/datasets/
+  NCIT: http://purl.obolibrary.org/obo/NCIT_
+  OMIABIS: http://purl.obolibrary.org/obo/OMIABIS_
+  OBI: http://purl.obolibrary.org/obo/OBI_
+  EFO: http://www.ebi.ac.uk/efo/EFO_ # I added this one
+  GSSO: http://purl.obolibrary.org/obo/GSSO_ # I added this one
+
+mappings:
+  fairgenomes:
+    sources:
+      - [samples.csv~csv] # I am going to name it like this samples.csv
+    s: ega.dataset:$(accession_id) # I am unsure for this part, I have not changed
+    po:
+      - [a, NCIT:C43376]
+      - [OMIABIS:0000006, ega.dataset:$(accession_id)~iri] #I am unsure for this part,too
+      - [NCIT:C93400, $(accesion_id)] # This is new. Are we going to use instead of the accesion_id over? 
+      - [EFO:0005518, $(description)] # This is new 
```

Comment:
I realise that it might be useful to add the labels for the terms as part of the comments, e.g. 
`      - [EFO:0005518, $(description)] # "sample collection protocol" - This is new`

Reply:
I can do it!

Reply:
Regarding this comment, I just noticed that FAIR Genomes has its own namespace and an ontology files that provides links to the external ontology terms. It can still be useful to add links to external ontologies explicitly since they then will be accessible in the resulting output file but instead of adding comments you could define a prefix  
`fgs: https://w3id.org/fair-genomes/ontology/` and refer to the corresponding terms as `- [a, fgs:Analysis]`. It is ok to repeat `- [a, …]` as long as it makes sense, e.g. the thing being describes “is a `fgs:Analysis`” and at the same time is “is a …”

Reply:
I did not understand this comment! I guess that you refer to the comments I wrote such as  # This is new. Or something different? I think we must discuss this in RL.

Reply:
We can discuss but in the meantime, this is an example:

```
  fgs: https://w3id.org/fair-genomes/ontology/

mappings:
  fairgenomes:
    sources:
      - [samples.csv~csv] # I am going to name it like this samples.csv
    s: ega.dataset:$(accession_id) # I am unsure for this part, I have not changed
    po:
      - [a, NCIT:C43376] # "Biologically-Derived Material"
      - [a, fgs:Material]
      …
      - [EFO:0005518, $(description)] # "sample collection protocol" - This is new 
      - [fgs:Material_Sampling_protocol, $(description)]
```

Reply:
I think that I have understood the example that you gave. I will add fgs to prefixes and then I will have an `a` with fgs. The other line ` - [fgs:Material_Sampling_protocol, $(description)] ` that you added I do not understand 100% how you are thinking, it seems that it has an association with the previous line ` - [EFO:0005518, $(description)] # "sample collection protocol" ` . If I understood correctly, you suggest to have the fgs in order to explain the comments that I will make in the previous line ... but am I going to have comments in the form # sample collection protocol and then the fgs...bla bla bla?

Reply:
Another comment in your example ... I guess that this `s: ega.dataset:$(accession_id) ` you forgot to change to ega.samples.